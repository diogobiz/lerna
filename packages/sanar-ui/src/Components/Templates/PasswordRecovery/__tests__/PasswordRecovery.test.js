import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESPasswordRecoveryTemplate from '../'

it('renders correctly', () => {
    const component = (
        <ESPasswordRecoveryTemplate>
            <h1>ESPasswordRecoveryTemplate</h1>
        </ESPasswordRecoveryTemplate>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
