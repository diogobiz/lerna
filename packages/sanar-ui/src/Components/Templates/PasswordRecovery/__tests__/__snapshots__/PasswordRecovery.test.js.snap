// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-password-recovery-template es-password-recovery-template--with-brand-header es-password-recovery-template--full-height"
>
  <div
    className="es-brand-header"
  >
    <img
      alt="Sanar Logo in brand header."
      src="{}"
    />
  </div>
  <div
    className="es-password-recovery-template__container-content"
  >
    <div
      className="es-password-recovery-template__container-content__content"
    >
      <div
        className="es-password-recovery-template__container-content__content__infos"
      >
        <img
          className="hide"
        />
        <div
          className="es-typography es-password-recovery-template__container-content__content__infos--title mb-md es-typography__heading es-typography__heading--h4"
          data-testid="es-typography"
        />
        <div
          className="es-typography es-password-recovery-template__container-content__content__infos--subtitle es-typography--subtitle2"
          data-testid="es-typography"
        />
      </div>
      <div
        className="es-password-recovery-template__container-content__content__actions"
      />
    </div>
  </div>
  <div
    className="es-password-recovery-template__footer"
  >
    <img
      alt="sanar-logo"
      src={
        Object {
          "process": [Function],
        }
      }
    />
  </div>
</div>
`;
